lms/templates/course.html

```diff
@@ -5,6 +5,10 @@
 from django.core.urlresolvers import reverse
 %>
 <%page args="course" expression_filter="h"/>
+<%
+    container_string = _("Starts: {date}")
```

Comment:
The name `container_string` doesn't have particular meaning in this context - at the top-level of the file.